templates/swagger/v1_json.tmpl

```diff
@@ -550,6 +552,7 @@
             "required": true
           },
           {
+            "description": "'visibility' can be \"public\", \"limited\" or \"private\"",
```

Comment:
Description should be set in the referenced schema
```suggestion
```

Reply:
Yeah, I model this off of other body descriptions as I never could figure out how to do it in the struct the body points to. Of course I modeled it after my own work on other end points, like https://try.gitea.io/api/swagger#/repository/repoCreateFile. It is meant to be able to describe other fields as needed, but I put the one in that I was adding to make it clear. If I click "Model" in that example, it doesn't explain anything.

Reply:
Actually now see that my comments in my other PR for Repo edit that comments in the scheme show up in swagger! (I didn't know about clicking on "Model" before). I should have done that this way for the link I also gave above. Going to fix that for both structs.

Reply:
Ok, all the swagger comments I have been incorrectly making are now moved their structs as seems best to do in the latest commit. Had to put an extra Note: in the struct header comment of CreateFileOption and UpdateFileOption about author and committer as it doesn't seem to get the comments for those fields since it points to another struct.